### PR TITLE
Changed feature images of homepage and features' page

### PR DESCRIPTION
**Pull request by dro-1**

### CODE REVIEW

**Review by geektutor — COMMENTED** [review]
In your assets/scripts/home.js, I can see that you changed the base Url? What URL is going to be showing?

File: assets/scripts/home.js [hunk 1]
```diff
@@ -17,26 +17,26 @@ let imagesArr = [
...
-  indicators.forEach(el => {
+  indicators.forEach((el) => {
     el.style.pointerEvents = "none";
     el.classList.remove("active");
   });
 
   curr.classList.add("active");
 
-  featureImg.src = `${BASE_URL}${imagesArr[i]}`;
+  featureImg.src = `${BASE_URL_DEV}${imagesArr[i]}`;
```
Comment (geektutor) [inline assets/scripts/home.js]: Why are we using Base URL dev here?

File: assets/scripts/home.js [hunk 2]
```diff
@@ -51,16 +51,18 @@ const startInterval = start => {
...
     clearInterval(intervalId);
-    indicators.forEach(indicator => {
+    indicators.forEach((indicator) => {
       indicator.style.pointerEvents = "none";
       indicator.classList.remove("active");
     });
     indicator.classList.add("active");
     featureImg.classList.add("reappear");
-    featureImg.src = `${BASE_URL}${imagesArr[i]}`;
+    featureImg.src = `${BASE_URL_DEV}${imagesArr[i]}`;
```
Comment (geektutor) [inline assets/scripts/home.js]: Why are we using the base_url_dev here?

File: templates/features.php [hunk 3]
```diff
@@ -12,11 +12,10 @@
       <div class="pd-features-hero-container">
         <div class="pd-features-hero-content">
           <h1>
-            Experience individual and joint wealth management options.
+            Experience individual and connected wealth management options.
```
Comment (geektutor) [inline templates/features.php:15]: Please change this text to the former text in the file. It should be joint not connected

File: templates/features.php [hunk 4]
```diff
@@ -37,14 +36,14 @@
           </p>
           <div class="pd-features-feature-items">
             <div class="pd-features-feature-item">
-              <img src="<?php echo get_stylesheet_directory_uri(); ?>/assets/images/features-selected.svg" alt="save" />
+              <img src="<?php echo get_stylesheet_directory_uri(); ?>/assets/images/features-selected.svg" alt="" />
```
Comment (geektutor) [inline templates/features.php:39]: Alt tag is important for screen readers. Why did you take it off?

File: assets/scripts/home.js (hunk 1 above)
Comment (dro-1) [reply assets/scripts/home.js]: Base URL is a constant value that holds the url to the wp theme
const BASE_URL =
  "https://vested.ng/new/wp-content/themes/vested/assets/images/";
 BASE_URL_DEV is the relative path we use when testing locally

File: assets/scripts/home.js (hunk 2 above)
Comment (dro-1) [reply assets/scripts/home.js]: I already changed it back to BASE_URL. That's prolly why this file is labeled outdated. Check the file itself and you'll see it's base url

File: templates/features.php (hunk 4 above)
Comment (dro-1) [reply templates/features.php:39]: They're icons so they shouldn't have as they have no actual relevance to the content

**Pull request merged**